Documentation/gettingstarted/encryption-ipsec.rst

```diff
@@ -17,24 +17,25 @@ as Cilium-managed host traffic, will be encrypted using IPsec. This guide uses
 Kubernetes secrets to distribute keys. Alternatively, keys may be manually
 distributed, but that is not shown here.
 
-Packets destined to the same node they were sent out of are not encrypted.
-This is an intended behavior as it would not provide any benefits because the
-raw traffic on the node can be seen.
+Packets are sent not encrypted when they are destined to the same node from
```

Comment:
Note, the changes in this file will conflict with #14940 CC @aditighag

Reply:
Oh that was February, I quite forgot about this PR. Fair enough, let's put mine on hold, I'll rebase once Aditi's merged. Thanks!

Reply:
@qmonnet PR #15965 duplicated this paragraph to the wireguard guide for consistency. Could you apply the same change there on this other guide as well?

Reply:
Yes, indeed. :) the PR was on hold because there were some encryption fixes underway. Thanks!